pkg/deploy/deploy.go

```diff
@@ -164,9 +165,18 @@ func deployGraph(ctx context.Context, configGraph *simple.DirectedGraph, clients
 
 		for _, root := range roots {
 			node := root.(graph.ConfigNode)
-			go func(ctx context.Context, node graph.ConfigNode) {
-				errChan <- deployNode(ctx, node, configGraph, clients, resolvedEntities)
-			}(context.WithValue(ctx, log.CtxKeyCoord{}, node.Config.Coordinate), node)
+			switch deployStrategy := api.NewAPIs()[node.Config.Coordinate.Type].DeployStrategy.(type) {
+			case api.SeqDeployStrategy:
```

Comment:
Upon closer inspection: Don't you currently make things sequential twice? 
When building the graph the fake dependencies are added, ensuring that configs aren't on the same level but in one chain. 

When deploying instead of firing off a go routine to deploy each node on the same level (the current roots), you now deploy inside the main routine of `deployGraph` and sleep in it.

I think the fake dependencies are always needed, else you won't get things into the same independent component. But then, wouldn't be enough to deploy them normally and just optional add the wait after deployment before returning on the error channel? Each 'root level' is synchronized via the error channel, so the wait duration should still effectively give us a delay before the next sequential config is deployed (as the depend on each other)